docs/sources/flow/reference/components/local.file.md

```diff
@@ -30,7 +30,7 @@ Name | Type | Description | Default | Required
 `filename` | `string` | Path of the file on disk to watch | | yes
 `detector` | `string` | Which file change detector to use (fsnotify, poll) | `"fsnotify"` | no
 `poll_frequency` | `duration` | How often to poll for file changes | `"1m"` | no
-`is_secret` | `bool` | Marks the file as containing a [secret][] | `false` | no
+`is_secret` | `bool` | Marks the file as containing a [secret](https://grafana.com/docs/agent/latest/flow/config-language/expressions/types_and_values/#special-types) | `false` | no
```

Comment:
Hey @ansomerv! We store the link below (line 94). Could you amend it there, and possibly using a relative link?

Thanks in advance!